tests/conftest.py

```diff
@@ -0,0 +1,23 @@
+# Copyright 2018 Amazon.com, Inc. or its affiliates. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License"). You
+# may not use this file except in compliance with the License. A copy of
+# the License is located at
+#
+#     http://aws.amazon.com/apache2.0/
+#
+# or in the "license" file accompanying this file. This file is
+# distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF
+# ANY KIND, either express or implied. See the License for the specific
+# language governing permissions and limitations under the License.
+import pytest
+
+
+@pytest.fixture(scope='module', params=["1.4", "1.4.1", "1.5", "1.5.0"])
```

Comment:
It would be great to cover the default case, e.g. pass special word like 'default' and in the test do not pass version at all.

Reply:
But it would run one of the versions again for every test.
We have a specific test for testing default value.

Reply:
That's true about the double run but I didn't see the special (integ) test for default value.

Reply:
https://github.com/aws/sagemaker-python-sdk/pull/82/files/bc748929fc3088d99d1e5c00787e3ddad1a3a51b#diff-8e5bb247678c2ede89b070ed8c1926f2R321 
and we have another similar one for mxnet